pkgs/applications/networking/cluster/kube3d/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "kube3d-${version}";
+  version = "1.1.0";
+
+  goPackagePath = "github.com/rancher/k3d";
+
+  src = fetchFromGitHub {
+    owner  = "rancher";
+    repo   = "k3d";
+    rev    = "v${version}";
+    sha256 = "180q7a95znpkhfqcaw3asqrq22r6ppw98qsggp2wfm746mllg5pc";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/rancher/k3d;
+    description = "A helper to run k3s (Lightweight Kubernetes. 5 less than k8s) in a docker container";
+    license = licenses.mit;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ ngerstle ];
```

Comment:
I'm happy to give over being maintainer to more active/invested individuals

Reply:
I think I will be able to commit more time over summer to this. And it happened to be that I am doing a lot of stuff around k8s these days.

Reply:
@kuznero would you like me to list you as the maintainer?

Reply:
Yep.